### DISCUSSION TITLE
How can I preview multiple pages of different views

### DISCUSSION THREAD

**Discussion by qinglin** [frangoteam/FUXA#1451] [Q&A] [answered]
Hi, Dear, We are about to embedded the mimic views we created using FUXA into the iframe our frontend project , since our project needs to load about 4-5 views, but fuxa‘s home can only preview one view, is there any way to load multiple different view into iframe, or we need to change the source code

**Comment by unocelli** [comment, marked as answer]
Hi, I am not sure I understood...you can use http://localhost:4200/view?name=[your view name] to access directly to the views

**Reply by MatthewReed303** [reply]
@unocelli maybe we add it via url so you can add view name to the url, this would work on the webserver with standard Fuxa port, so you can still use DNS with reverse proxy etc `localhost:1881/home/viewname` the main project still runs under just home, but if you have another external application, you can display a single Fuxa view in an iframe on a website etc

For example, I have been asked to embed a single Fuxa view into another application and would do this via iframe and dns and reverse proxy on the main Fuxa app

**Reply by qinglin** [reply]
Thanks!! That's exactly what I need !

**Reply by unocelli** [reply]
Hi, are there problems with passing the view name in the url as a parameter?

**Reply by MatthewReed303** [reply]
It does work on main server and proxy, just got to remove [ ] brackets, for example localhost:1881/view?name=MainView

**Reply by unocelli** [reply]
Sorry I should describe with <> where I intend that is mandatory, in any case you have to remove it and use localhost:1881/view?name=MainView
